experimental/dds/tree2/src/test/feature-libraries/editable-tree-2/objectFactory.spec.ts

```diff
@@ -195,215 +195,179 @@ describe("SharedTreeObject factories", () => {
-			objectMap: comboSchemaBuilder.map(comboChild),
-			listList: comboSchemaBuilder.list(comboSchemaBuilder.list(comboChild)),
-			listMap: comboSchemaBuilder.map(comboSchemaBuilder.list(comboChild)),
-			mapList: comboSchemaBuilder.list(comboSchemaBuilder.map(comboChild)),
-			mapMap: comboSchemaBuilder.map(comboSchemaBuilder.map(comboChild)),
+			content: [
+				comboChild,
+				comboSchemaBuilder.list(comboChild),
+				comboSchemaBuilder.map(comboChild),
+			],
+		});
+		const comboRoot = comboSchemaBuilder.object("Root", {
+			id: comboSchemaBuilder.number,
+			content: [
+				comboParent,
+				comboSchemaBuilder.list(comboParent),
+				comboSchemaBuilder.map(comboParent),
+			],
 		});
 		const comboSchema = comboSchemaBuilder.intoSchema(
 			// TODO: This extra root won't be necessary once the true root of the tree is settable
-			comboSchemaBuilder.object("root", { root: comboSchemaBuilder.optional(comboParent) }),
+			comboSchemaBuilder.object("root", { root: comboSchemaBuilder.optional(comboRoot) }),
 		);
 
+		type ComboRoot = ProxyNode<typeof comboRoot>;
 		type ComboParent = ProxyNode<typeof comboParent>;
 		type ComboChild = ProxyNode<typeof comboChild>;
 		type ComboLeaf = ProxyNode<typeof comboLeaf>;
-		type ComboObject = ComboParent | ComboChild | ComboLeaf;
+		type ComboObject = ComboRoot | ComboParent | ComboChild | ComboLeaf;
 
-		/** Iterates through all the objects in the combo tree */
+		/** Iterates through all the objects in a combo tree */
 		function* walkComboObjectTree(object: ComboObject): IterableIterator<ComboObject> {
 			yield object;
-			if ("object" in object) {
-				yield* walkComboObjectTree(object.object);
-				for (const item of object.objectList) {
-					yield* walkComboObjectTree(item);
-				}
-				for (const value of object.objectMap.values()) {
-					yield* walkComboObjectTree(value);
-				}
-				for (const list of object.listList) {
-					for (const item of list) {
-						yield* walkComboObjectTree(item);
-					}
-				}
-				for (const list of object.listMap.values()) {
-					for (const item of list) {
-						yield* walkComboObjectTree(item);
-					}
-				}
-				for (const map of object.mapList) {
-					for (const item of map.values()) {
-						yield* walkComboObjectTree(item);
+			if ("content" in object) {
+				const { content } = object;
+				if (content instanceof Map) {
+					for (const value of content.values()) {
+						yield* walkComboObjectTree(value);
 					}
-				}
-				for (const map of object.mapMap.values()) {
-					for (const item of map.values()) {
+				} else if (Array.isArray(content)) {
+					for (const item of content) {
 						yield* walkComboObjectTree(item);
 					}
+				} else {
+					yield* walkComboObjectTree(content as ComboObject);
 				}
 			}
 		}
 
 		/**
-		 * Builds trees of {@link ComboObject}s.
-		 * Records all built objects and assigns each a unique ID.
+		 * Defines the structure of a combo tree.
+		 * @example A layout of
```

Comment:
Nit: The `@example` tag has specific requirements regarding its first line. TSDoc spec calls for the first line to be treated as the "title" of the example if present. So in a case where we're just starting with a statement like this, it would be better to do:
```suggestion
		 * @example
		 * A layout of
```

I learned about this somewhat recently and am trying to raise awareness :) I should make a post on Teams...

https://github.com/microsoft/FluidFramework/wiki/TSDoc-Guidelines#example-titles

Reply:
Oh interesting. Thanks for the heads up. Are there any other tags with similar rules?

Reply:
I don't believe so.